adhocracy4/modules/models.py

```diff
@@ -25,6 +32,32 @@ def settings_instance(self):
                 return getattr(self, setting)
 
     @property
+    def is_active(self):
+        return self == self.project.last_active_module
```

Comment:
I don't like the wording here. For me, a module is "active" when it has an active phase. This also how we use the term in the meinBerlin frontend. I have some ideas for alternatives:

- In the frontend, we also have the concept of a "selected" module (i.e. the module that the user is currently looking at). So maybe `is_selected_by_default`. (I also think that this property is only relevant for UI)

- Maybe call it by what it is: `is_last_active`.

- Maybe just leave it out completely. `module == project.last_active_module` is not that much to type and perfectly understandable..